frontend/packages/knative-plugin/src/topology/components/anchors/EventSinkSourceAnchor.ts

```diff
@@ -0,0 +1,12 @@
+import { AbstractAnchor, Point } from '@patternfly/react-topology';
+
+export default class EventSinkSourceAnchor extends AbstractAnchor {
+  getLocation(): Point {
+    return this.getReferencePoint();
+  }
+
+  getReferencePoint(): Point {
+    const bounds = this.owner.getBounds();
+    return new Point(bounds.x, bounds.y + bounds.height / 2);
```

Comment:
this is sink just we can have as below

```suggestion
    return new Point(bounds.right(), bounds.y + bounds.height / 2);
```